fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Blob.kt

```diff
@@ -3,7 +3,7 @@ package com.github.kittinunf.fuel.core
 import java.io.InputStream
 
 data class Blob(
-    val name: String = "",
+    val name: String,
```

Comment:
out of interest why did you remove this default arg?

Reply:
Haven't figured out completely how I think this API should work, but mostly because `name` doesn't exist on request anymore. Explicit naming works much better. There are already a few tests generating `_0` as name, which does not make any sense, to me.

Removing this default, but allowing some sort of "default options" via the request and injecting it via `blob` would work.